medical_pathology/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Medical Pathology",
+    "summary": "Extends Odoo Medical with pathologies (diseases).",
+    "version": "10.0.1.0.0",
+    "category": "Medical",
+    "website": "https://laslabs.com/",
```

Comment:
In here it should be: "https://github.com/OCA/vertical-medical" as mentioned in https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L9